app/views/idv/shared/_ssn.html.erb

```diff
@@ -28,6 +28,17 @@ locals:
   <%= new_window_link_to(t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url) %>
 </p>
 
+<% if IdentityConfig.store.proofing_device_profiling_collecting_enabled %>
+  <% unless IdentityConfig.store.lexisnexis_threatmetrix_org_id.empty? || updating_ssn %>
+    <script type="text/javascript" src="https://h.online-metrix.net/fp/tags.js?org_id=<%= IdentityConfig.store.lexisnexis_threatmetrix_org_id %>&session_id=<%= flow_session[:threatmetrix_session_id] %>">
```

Comment:
Questions, maybe we can address in a follow-up PR later, but:

- Do we need to worry about adding a nonce to this `<script>` tag so our CSP will allow it in prod? ex 
- What if we used our `URIService.add_params` helper so we don't have to manually interpolate?

```suggestion
    <script type="text/javascript" src="<%= URIService.add_params("https://h.online-metrix.net/fp/tags.js", org_id:  IdentityConfig.store.lexisnexis_threatmetrix_org_id, session_id: flow_session[:threatmetrix_session_id]) %>">
```

Reply:
Another thought, we may want to turn the domain into a config value but that's not needed right now